charts/zookeeper-operator/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: zookeeper-operator
+version: 0.2.2
+appVersion: 0.2.2
+description: zookeeper operator Helm chart for Kubernetes
+keywords:
+- zookeeper
+- storage
+home: https://github.com/pravega/charts/blob/master/zookeeper-operator
```

Comment:
This should point to charts under zookeeper-operator

Reply:
done